ami/jobs/management/commands/chaos_monkey.py

```
@@ -0,0 +1,97 @@
+                for stream in streams:
+                    name = stream.config.name
+                    await js.delete_stream(name)
+                    deleted.append(name)
+                return deleted
+            finally:
+                await nc.close()
+
+        try:
+            deleted = async_to_sync(_delete_all_streams)()
+        except Exception as e:
+            raise CommandError(f"Failed to flush NATS: {e}") from e
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)